CHANGELOG.rst

```diff
@@ -1,7 +1,13 @@
 CHANGELOG
 =========
 
-4.0.0.6 (2020-03-17)
+4.0.0.12 (2020-04-29)
+--------------------
```

Comment:
there is one - missing

Reply:
@FinalAngel I'm not following, what is missing?

Reply:
instead of:

```
4.0.0.12 (2020-04-29)
--------------------
```

it should be

```
4.0.0.12 (2020-04-29)
---------------------
```

in order for the dashes to align with the headline

Reply:
Ok, dashes, NP, will fix now.